content/includes/top-nav.html

```diff
@@ -28,6 +28,7 @@
     <div role="banner" class="va-header-logo-wrapper">
       <a href="/" class="va-header-logo" onClick="recordEvent({ event: 'nav-header-logo' });">
       {% include "assets/img/svglogo.svg" %}
+      <span class="usa-sr-only">Vets dot gov from the U S Department of Veteran Affairs</span>
```

Comment:
@ncksllvn Is there a good way to flag this for change / removal after the brand consolidation effort gets merged in?
Maybe like a
```
{% comment %} brand consolidation change needed {% endcomment %}
```
or something regular and searchable like that? My hope is that we can search for this one string through the whole project to find all the things we've already identified need to change.

Reply:
Good idea! I'm up for defining that somehow. We can update the [doc](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/platform/brand-consolidation) to reflect that too.

Reply:
Follow-up on this - I don't think there's a special comment needed anymore, because VA.gov has its own Metalsmith directory (`va-gov`) that is now on master. If there's something going into Vets.gov content that you want to also see included in VA.gov content, then the change should go into the `va-gov` directory as well